<html>
 <head></head>
 <body>
  <p>There is the following statement inside a method:</p>
  <pre><code>throw new Exception();</code></pre>
  <p>Select the correct possible declaration of this method.</p>
  <p><button onclick="getElementById('hint-195').style.display='inline'" data-gtm-vis-recent-on-screen-10745337_362="40614" data-gtm-vis-first-on-screen-10745337_362="40614" data-gtm-vis-total-visible-time-10745337_362="100" data-gtm-vis-has-fired-10745337_362="1" data-gtm-vis-recent-on-screen-10745337_383="40620" data-gtm-vis-first-on-screen-10745337_383="40620" data-gtm-vis-total-visible-time-10745337_383="100" data-gtm-vis-has-fired-10745337_383="1"> Hint </button></p>
  <div id="hint-195" style="display:none;">
   Remember, creating and throwing exceptions are not the same action!
  </div>
 </body>
</html>